### PR TITLE
Migrate TwineAuthenticateV0-1 to Node 10

### PR DESCRIPTION
**Pull request by AndreyIvanov42**
**Task name**: TwineAuthenticateV0-1

**Description**: migrate task to Node 10

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Comment by AndreyIvanov42** [comment]
Tested with pipeline for [canary builds ](https://github.com/microsoft/azure-pipelines-canary/blob/master/src/packaging/TwineAuthenticate/twine-authenticate.yml)

**Comment by anatolybolshakov** [comment]
@AndreyIvanov42 please check failed tests - can we fix it quickly or it would require some investigation of how task works?

**Comment by AndreyIvanov42** [comment]
@anatolybolshakov Some investigation is need

**Comment by AndreyIvanov42** [comment]
@anatolybolshakov
I increased timeout and this helped. PR is ready currently

**Comment by AndreyIvanov42** [comment]
@geekzter

**Review by EzzhevNikita — APPROVED** [review]

**Comment by AndreyIvanov42** [comment]
@phil-hodgson @aasim @satbai Could you please review and approve the PR if all is fine?

**Comment by DmitriiBobreshev** [comment]
Test results are [here](https://github.com/microsoft/build-task-team/issues/3193)

Risks analysis checklist
- [x] There are no risky dependency updates
- [x] Changes have been tested
- [x] Enough test coverage for changes and current test coverage for the task doesn't look poor
- [x] We understand how tasks are working, how changes affect task behavior
- [x] There are no breaking changes
- [x] There are no other concerns
- [ ] I have not discovered any new uncovered test/use cases

File: Tasks/TwineAuthenticateV1/task.json [hunk 1]
```diff
@@ -9,14 +9,14 @@
     "category": "Package",
     "version": {
         "Major": 1,
-        "Minor": 198,
+        "Minor": 201,
```
Comment (KonstantinTyukalov) [inline Tasks/TwineAuthenticateV1/task.json:12]: Should it be `207` instead?

**Review by tintse-thxsky-MSFT — APPROVED** [review]

File: Tasks/TwineAuthenticateV1/task.json (hunk 1 above)
Comment (mpodriezov) [reply Tasks/TwineAuthenticateV1/task.json]: Yes, 208 even at this moment

**Pull request merged**